webhook.rb

```diff
@@ -0,0 +1,42 @@
+require 'sinatra'
+require 'sinatra/namespace'
+
+components = %w[barong peatio frontend tower]
+
+namespace '/deploy' do
+  before do
+    content_type 'application/json'
+  end
+
+  get '/ping' do
+    'pong'
+  end
+
+  post '/' do
```

Comment:
get will be easier to call I believe
maybe /deploy/:payload

the payload being a JWT